cvpy/annotation/cvat/CVATProject.py

```diff
@@ -210,8 +210,41 @@ def resume(project_name: str, cas_connection: CAS, caslib: str, relative_path: s
             SPecifies the caslib under which CAS tables were saved.
```

Comment:
If you submit another patchset, would you mind fixing "SPecifies"?

Reply:
Good catch. I fixed it.